assets/app/views/builds.html

```diff
@@ -100,7 +100,7 @@
                         <span ng-if="build.spec.source.type == 'None'">
                           <i>none</i>
                         </span>
-                        <span ng-if="build.spec.source.type == 'Git'" ng-bind-html='build.spec.source.git.uri | githubLink : build.spec.source.git.ref : build.spec.source.contextDir | linky'></span>
+                        <span class="word-break" ng-if="build.spec.source.type == 'Git'" ng-bind-html='build.spec.source.git.uri | githubLink : build.spec.source.git.ref : build.spec.source.contextDir | linky'></span>
```

Comment:
@spadgett without it it willl overflow
![screenshot-6](https://cloud.githubusercontent.com/assets/1668218/15047102/894a76b4-12e3-11e6-9c3e-ff8d64145334.png)